2021-12-13 17:49:25,091 INFO    MainThread:8644 [wandb_setup.py:_flush():71] setting env: {}
2021-12-13 17:49:25,091 INFO    MainThread:8644 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-13 17:49:25,092 INFO    MainThread:8644 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20211213_174925-3ajhx5nj\logs\debug.log
2021-12-13 17:49:25,092 INFO    MainThread:8644 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20211213_174925-3ajhx5nj\logs\debug-internal.log
2021-12-13 17:49:25,092 INFO    MainThread:8644 [wandb_init.py:init():404] calling init triggers
2021-12-13 17:49:25,093 INFO    MainThread:8644 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-12-13 17:49:25,094 INFO    MainThread:8644 [wandb_init.py:init():449] starting backend
2021-12-13 17:49:25,094 INFO    MainThread:8644 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2021-12-13 17:49:25,102 INFO    MainThread:8644 [backend.py:ensure_launched():199] starting backend process...
2021-12-13 17:49:25,232 INFO    MainThread:8644 [backend.py:ensure_launched():204] started backend process with pid: 8568
2021-12-13 17:49:25,233 INFO    MainThread:8644 [wandb_init.py:init():458] backend started and connected
2021-12-13 17:49:25,236 INFO    MainThread:8644 [wandb_init.py:init():519] updated telemetry
2021-12-13 17:49:25,270 INFO    MainThread:8644 [wandb_init.py:init():549] communicating current version
2021-12-13 17:49:26,257 INFO    MainThread:8644 [wandb_init.py:init():554] got version response 
2021-12-13 17:49:26,257 INFO    MainThread:8644 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-13 17:49:26,577 INFO    MainThread:8644 [wandb_init.py:init():592] starting run threads in backend
2021-12-13 17:49:26,715 INFO    MainThread:8644 [wandb_run.py:_console_start():1816] atexit reg
2021-12-13 17:49:26,717 INFO    MainThread:8644 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2021-12-13 17:49:26,717 INFO    MainThread:8644 [wandb_run.py:_redirect():1727] Wrapping output streams.
2021-12-13 17:49:26,722 INFO    MainThread:8644 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-13 17:49:26,722 INFO    MainThread:8644 [wandb_init.py:init():619] run started, returning control to user process
2021-12-13 17:49:26,724 INFO    MainThread:8644 [wandb_config.py:__setitem__():144] config set learning_rate = 1e-05 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001AA8AB55DC0>>
2021-12-13 17:49:26,724 INFO    MainThread:8644 [wandb_run.py:_config_callback():962] config_cb learning_rate 1e-05 None
2021-12-13 17:49:26,727 INFO    MainThread:8644 [wandb_config.py:__setitem__():144] config set batch_size = 10 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001AA8AB55DC0>>
2021-12-13 17:49:26,727 INFO    MainThread:8644 [wandb_run.py:_config_callback():962] config_cb batch_size 10 None
2021-12-13 17:49:28,633 INFO    MainThread:8644 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-12-13 17:49:28,634 INFO    MainThread:8644 [wandb_run.py:_restore():1758] restore
2021-12-13 17:49:29,284 INFO    MainThread:8644 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 17:49:29,596 INFO    MainThread:8644 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 17:49:30,619 INFO    MainThread:8644 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 3643
}

2021-12-13 17:49:30,730 INFO    MainThread:8644 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 3669
}

2021-12-13 17:49:30,847 INFO    MainThread:8644 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 3669
}

2021-12-13 17:49:30,961 INFO    MainThread:8644 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3669
  total_bytes: 3669
}

2021-12-13 17:49:31,068 INFO    MainThread:8644 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3669
  total_bytes: 3669
}

2021-12-13 17:49:31,181 INFO    MainThread:8644 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3669
  total_bytes: 3669
}

2021-12-13 17:49:31,295 INFO    MainThread:8644 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3669
  total_bytes: 3669
}

2021-12-13 17:49:31,403 INFO    MainThread:8644 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3669
  total_bytes: 3669
}

2021-12-13 17:49:31,524 INFO    MainThread:8644 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3669
  total_bytes: 3669
}

2021-12-13 17:49:31,722 INFO    MainThread:8644 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3669
  total_bytes: 3669
}

2021-12-13 17:49:32,243 INFO    MainThread:8644 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3669
  total_bytes: 3669
}
local_info {
}

2021-12-13 17:49:33,310 INFO    MainThread:8644 [wandb_run.py:_append_files():2184] logging synced files
